pages/common/7zr.md

```diff
@@ -1,17 +1,8 @@
-# 7zr
+# 7za
```

Comment:
Was that intentional?
```suggestion
# 7zr
```

Reply:
No, it most certainly was not.  I copied the contents of `7za.md` :smile: